eos-tech-support/eos-list-users

```diff
@@ -1,27 +1,41 @@
 #!/bin/bash -e
 
-# Lists all users (i.e., all home directories on /dev/sda2)
-
-DEVICE=/dev/sda2
-MOUNT=/mnt/sda2
+# Lists all users (i.e., all home directories on all ostree devices)
 
 USERID=$(id -u)
 if [ "$USERID" != "0" ]; then
     echo "Program requires superuser privileges"
     exit 1
 fi
 
+MOUNT=`mktemp -d`
+
 cleanup() {
     set +e
-    umount $MOUNT
+    if mount | grep $MOUNT; then
```

Comment:
Yes, I thought of that, but if the umount fails for some other reason, we would not see the error message.  On the other hand, perhaps there is no practical other reason why umount would fail that we would care about, but still I think it is better to not hide potentially real error messages.